be/src/util/jvm_metrics.cpp

```diff
@@ -90,9 +92,12 @@ JvmMetrics::JvmMetrics(MetricRegistry* registry, JNIEnv* env) {
             break;
         }
         try {
-            _jvm_stats.init(env);
+            Status st = _jvm_stats.init();
+            if (!st) {
+                LOG(WARNING) << "jvm Stats Init Fail. " << st.to_string();
```

Comment:
Should we call `break` if `!st`?

Reply:
It doesn't matter if it is needed or not, because I will check `_jvm_stats.init_complete()`.